adapter.go

```diff
@@ -143,7 +143,7 @@ func NewAdapter(driverName string, dataSourceName string, params ...interface{})
 		} else {
```

Comment:
What do you mean by `lock method UpdatePolicies`? Is it `add method UpdatePolicies`?